### PR TITLE
fix(processor): remove duplicate preprocessing steps in XVLA LIBERO pipeline

### LINKED ISSUE #2955: Duplicated Steps in Environment and Policy PreProcessor when evaluating X-VLA on LIBERO
### Ticket Type

🐛 Bug Report (Something isn't working)

### Environment & System Info

```Shell

```

### Description

Note that this issue coincides heavily with #2954.

I identified a redundancy in the preprocessing pipeline when evaluating the **XVLA policy** within the **LIBERO environment**. 

The pipeline instantiates two processor chains:
1. **Environment Preprocessor:** via `make_env_pre_post_processors` → `make_xvla_libero_pre_post_processors`
2. **Policy Preprocessor:** via `make_pre_post_processors` → `make_xvla_pre_post_processors`

Both chains currently append `XVLAImageNetNormalizeProcessorStep` and `XVLAAddDomainIdProcessorStep`. Because the pipeline executes `Env Preprocessor` → `Policy Preprocessor`, these transformations are applied twice to the input observations.

### Reproduction / Code Analysis
The duplication occurs in `lerobot/policies/xvla/processor_xvla.py`.

**1. Environment-Specific Processor (`make_xvla_libero_pre_post_processors`)**
Adds normalization and domain ID:
```python
def make_xvla_libero_pre_post_processors() -> tuple[...]:
    # ...
    pre_processor_steps.extend(
        [LiberoProcessorStep(), XVLAImageNetNormalizeProcessorStep(), XVLAAddDomainIdProcessorStep()]
    )
    # ...
```

2. General Policy Processor (make_xvla_pre_post_processors) Also adds normalization and domain ID:
```python
def make_xvla_pre_post_processors(...) -> tuple[...]:
    # ...
    input_steps = [
        # ...
        XVLAImageToFloatProcessorStep(),
        XVLAImageNetNormalizeProcessorStep(), # DUPLICATE
        XVLAAddDomainIdProcessorStep(),       # DUPLICATE
        # ...
    ]
```
This leads to double normalization which in turns leads to erroneous input values and redundant processing. 

### Suggested Fix
Remove the general XVLA steps from the environment-specific processor (make_xvla_libero_pre_post_processors). It should likely only handle LiberoProcessorStep and leave the standard normalization to the policy processor.

```python
def make_xvla_libero_pre_post_processors() -> tuple[...]:
    # ...
    # Removed XVLAImageNetNormalizeProcessorStep and XVLAAddDomainIdProcessorStep
    pre_processor_steps.extend(
        [LiberoProcessorStep()]
    )
    # ...
```

### Additional Remarks
Currently this issue is mitigated when directly evaluating the pre-trained lerobot/xvla-libero checkpoint available on huggingface as the checkpoint there does not include the `XVLAImageToFloatProcessorStep` and the `XVLAImageNetNormalizeProcessorStep` (which I think is incorrect, see #2954).
When I first finetuned lerobot/xvla-libero on LIBERO with the additional `XVLAImageToFloatProcessorStep` and the `XVLAImageNetNormalizeProcessorStep`  steps in the preprocessing pipeline and then evaluated this model I ran into the double normalization error which led to negative input values.

### Context & Reproduction

_No response_

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

### Additional Info / Workarounds

_No response_

### PR DESCRIPTION
**Pull request by jashshah999**
## Summary

- Removed `XVLAImageNetNormalizeProcessorStep` and `XVLAAddDomainIdProcessorStep` from `make_xvla_libero_pre_post_processors`
- These steps are already applied by the policy-level preprocessor in `make_xvla_pre_post_processors`, so they were being executed twice (env preprocessor then policy preprocessor), causing double ImageNet normalization and producing erroneous negative input values

## Test plan

- Verified the policy preprocessor (`make_xvla_pre_post_processors`) already includes both `XVLAImageNetNormalizeProcessorStep` and `XVLAAddDomainIdProcessorStep`
- The env-level processor should only handle LIBERO-specific observation formatting (`LiberoProcessorStep`), not general XVLA preprocessing

Fixes #2955

### CODE REVIEW

**Comment by jashshah999** [comment]
Friendly ping — just checking if this is ready for review when you get a chance. Happy to address any feedback!
